lms/djangoapps/django_comment_client/management/commands/rename_user.py

```diff
@@ -0,0 +1,22 @@
+"""
+Change the username of an existing user
+"""
+
+from django.core.management.base import BaseCommand, CommandError
+from .. import management_utils
+
+
+class Command(BaseCommand):
+    """
+    invoke this manage.py command from the console as follows:
+    python manage.py lms rename_user <username1> <username2> --settings=devstack
+    """
+    args = '<old_username> <new_username>'
+    help = 'Modify the username of an existing user'
+
+    def handle(self, *args, **options):
+        if len(args) != 2:
+            raise CommandError('Usage is rename_user {0}'.format(self.args))
+        result = management_utils.rename_user(*args)
+        if (result['status'] == 'success'):
```

Comment:
Changed the hash to  have key 'success' and value True/False, since there are only two possible values. We only print a message on success because, in all other cases, an exception is raised and printed using log.exception. These exceptions are thrown from the management_utils rename_user method. As per Jason's suggestion this will allow us to use the rename_command not just as a command line tool, but also from the UI in the future.